spec/observables/forkJoin-spec.ts

```diff
@@ -595,5 +611,164 @@ describe('forkJoin', () => {
         },
       });
     });
+
+    it('concat reference', () => {
+      const results: Array<number | string> = [];
+      concat(of(1), of(2))
+        .pipe(last())
+        .subscribe({
+          next(value) {
+            results.push(+value);
```

Comment:
Interesting. Why do we need to convert `value` to a number here? It should already be a number?